### PR TITLE
Add Kubernetes Python example

### PR DESCRIPTION
**Pull request by aareet**
Inspired by #185, a Kubernetes example in Python based on the config from the [HashiCorp Kubernetes Learn guide](https://learn.hashicorp.com/terraform/kubernetes/deploy-nginx-kubernetes).

### CODE REVIEW

File: examples/python-kubernetes/README.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+The above command will create a folder called `cdktf.out` that contains all Terraform JSON configuration that was generated.
+
+Run Terraform commands
+
+```bash
+cd cdktf.out
+terraform init
+terraform plan
+terraform apply
+```
```
Comment (anubhavmishra) [inline examples/python-kubernetes/README.md:43]: I would also add a section that uses `cdktf deploy` command to deploy the example.

Comment (aareet) [reply examples/python-kubernetes/README.md:48]: Thanks! Added a cdktf deploy section and squashed my commits

File: examples/python-kubernetes/Pipfile [hunk 2]
```diff
@@ -0,0 +1,10 @@
+[[source]]
+name = "pypi"
+url = "https://pypi.org/simple"
+verify_ssl = true
+
+[requires]
+python_version = "3.7"
+
+[packages]
+cdktf = "~=0.0.12"
```
Comment (skorfmann) [inline examples/python-kubernetes/Pipfile:10]: Please use the references [as here](https://github.com/hashicorp/terraform-cdk/blob/b8aeded4cb58a7f367195d5ed8a4632588f43eca/examples/python-aws/Pipfile)

```
[[source]]
name = "pypi"
url = "https://pypi.org/simple"
verify_ssl = true

[requires]
python_version = "3.7"

[packages]
cdktf = {path = "./../../dist/python/cdktf-0.0.0-py3-none-any.whl"}
```

Comment (aareet) [reply examples/python-kubernetes/Pipfile]: Made the change 🍰

**Pull request merged**